src/ansible_navigator/configuration_subsystem/configurator.py

```diff
@@ -234,7 +234,17 @@ def _apply_cli_params(self) -> None:
                 self._messages.append(LogMessage(level=logging.INFO, message=message))
 
     def _post_process(self) -> None:
+        delayed = []
+        normal = []
+
+        # Sort delayed and normal entries.
```

Comment:
Would this be more precise?
```suggestion
        # Separate delayed and normal entries.
```